### PR TITLE
Judicial Markers explode faster

### PR DESCRIPTION
**Pull request by ChangelingRain**
:cl: Joan
tweak: Judicial Markers(the things spawned from Ratvar's Flame, in turn spawned from a Judicial Visor) explode one second faster.
/:cl:
Slightly faster aoe stun, ahoy

### CODE REVIEW

**Comment by InsaneHyena** [comment]
With Ratvar stun not being useless anymore, I think we're entering the "Ratvar is OP" territory. If played properly, it's much more powerful than regular cult.

**Comment by KorPhaeron** [comment]
We currently have 4 clock buffs open yeah

**Comment by ChangelingRain** [comment]
Don't worry, there'll be a nerf of a sort up shortly.

File: code/game/gamemodes/clock_cult/clock_structures.dm [hunk 1]
```diff
@@ -427,10 +427,10 @@
 /obj/effect/clockwork/judicial_marker/New()
 	..()
 	flick("judicial_marker", src)
-	spawn(25) //Utilizes spawns due to how it works with Ratvar's flame
+	spawn(16) //Utilizes spawns due to how it works with Ratvar's flame
```
Comment (coiax) [inline code/game/gamemodes/clock_cult/clock_structures.dm]: reeeeeee

Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_structures.dm]: can't do addtimer because it needs accuracy, not 'once every half a second'

Comment (coiax) [inline code/game/gamemodes/clock_cult/clock_structures.dm]: Stoned MC has 2ds accuracy.

Comment (ChangelingRain) [inline code/game/gamemodes/clock_cult/clock_structures.dm]: I dislike trying to sync sprites with things that update slower than the sprites, so I'd really rather not, especially if it turns out I have to change this again.

**Pull request merged**